### PR TITLE
[EASY] Tutorial web updates

### PR DESCRIPTION
**Pull request by vincentschen**

### CODE REVIEW

File: .web.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 tutorials:
-  - title: Snorkel Intro Tutorial
+  - title: Introduction to Labeling Functions
```
Comment (paroma) [inline .web.yml]: do we want "Introduction to" or "Intro to"? And we should be consistent. I vote "Intro to", fits in the website boxes better

File: .web.yml [hunk 2]
```diff
@@ -8,26 +8,26 @@ tutorials:
...
     notebook: visual_relation
     exclude_output: true
   - title: Information Extraction in Snorkel
-    description: Labeling spouse mentions in sentences
+    description: Labeling spouse mentions in documents
     notebook: spouse
     exclude_output: true
   - title: Introduction to Snorkel's Multitask Learning System
-    description: State-of-the-art pretraining and task flows
+    description: State-of-the-art framework for multitask learning (i.e. pretraining, parameter sharing)
```
Comment (paroma) [inline .web.yml]: again for website, leaving these short and sweet looks better on the landing page, but we can try to see how to fix that

File: .web.yml (hunk 1 above)
Comment (vincentschen) [reply .web.yml]: yes you're right

File: .web.yml (hunk 2 above)
Comment (vincentschen) [reply .web.yml]: updated !

**Pull request merged**